backend/src/Notifo.Domain.Integrations/Discord/DiscordServiceExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// =====================================================
+//  Notifo.io
+// ==========================================================================
+//  Copyright (c) Sebastian Stehle
+//  Author of the file: Artur Nowak
```

Comment:
I think the header is fixed and it should give a compiler warning if it is not the correct one. You can add yourself as contributor to the readme and we can also modify the header, but we cannot change it on a per-file basis.